tools/src/test/java/org/apache/kafka/tools/TopicCommandIntegrationTest.java

```diff
@@ -472,10 +516,11 @@ public void testCreateAlterTopicWithRackAware(String quorum) throws Exception {
             "--topic", testTopicName);
         topicService.alterTopic(alterOpts);
 
+        TestUtils.waitForAllReassignmentsToComplete(adminClient, 100L);
         kafka.utils.TestUtils.waitUntilTrue(
             () -> brokers().forall(p -> p.metadataCache().getTopicPartitions(testTopicName).size() == alteredNumPartitions),
             () -> "Timeout waiting for new assignment propagating to broker",
-            org.apache.kafka.test.TestUtils.DEFAULT_MAX_WAIT_MS, 100L);
+            org.apache.kafka.test.TestUtils.DEFAULT_MAX_WAIT_MS, 500L);
```

Comment:
Why changing `pause` from 100ms to 500ms?

Reply:
It turns out that the flakiness of the `testDescribeUnderReplicatedPartitionsWhenReassignmentIsInProgress` is not due to the `DEFAULT_MAX_WAIT_MS`. So I've reverted this change. Thanks ;)